examples/request_header_identity_provider_vars.yaml

```diff
@@ -0,0 +1,9 @@
+---
+
+# Request-header identity provider vars 
+request_header_provider: false  # Set true to enable the test execution
+request_header_user_display_name: ""  # User's full name to be displayed e.g. "John Doe"
+request_header_username: ""  # Username to be attached in the request header e.g. "jdoe" 
+ca_configmap_name: ""  # Name for a ConfigMap object containing CA certificate bundle
```

Comment:
Can we set the default values in the example vars

Reply:
I have been set the default values to the role variables in `examples/` vars.